common/src/test/java/io/seata/common/loader/EnhancedServiceLoaderTest.java

```diff
@@ -98,4 +98,32 @@ public void getAllExtensionClass1() {
         assertThat(allExtensionClass).isNotEmpty();
     }
 
+    @Test
+    public void getSingletonExtensionInstance(){
+        Hello hello1 = EnhancedServiceLoader.load(Hello.class, "ChineseHello");
+        Hello hello2 = EnhancedServiceLoader.load(Hello.class, "ChineseHello");
+        assertThat(hello1 == hello2);
+    }
+
+    @Test
+    public void getMultipleExtensionInstance(){
+        Hello hello1 = EnhancedServiceLoader.load(Hello.class, "LatinHello");
+        Hello hello2 = EnhancedServiceLoader.load(Hello.class, "LatinHello");
+        assertThat(hello1 != hello2);
+    }
+
+    @Test
+    public void getAllInstances(){
+        List<Hello> hellows1 = EnhancedServiceLoader.loadAll(Hello.class);
+        List<Hello> hellows2 = EnhancedServiceLoader.loadAll(Hello.class);
+        for (Hello hello : hellows1){
+            if(hello.say()!="Olá."){
+                assertThat(hellows2.contains(hello));
+            }
+            else{
+                assertThat(!hellows2.contains(hello));
+            }
+        }
+    }
```

Comment:
Wrong usage of `assertThat`, should be `assertThat(xxxx).isTrue()` or `assertThat(xxxx).isFalse()`.

Reply:
Changed to the right usage